content/en/apps/examples/supervisor-reference-app.md

```diff
@@ -0,0 +1,72 @@
+
+{{< figure src="Forms_hierarchy.png"  link="Forms_hierarchy.png" alt="Forms hierarchy" title="" class="left col-12">}}
+
+## Supervisor app workflows and features.
+
+### CHW monthly meeting workflows.
+A Supervisor conducts a CHW monthly meeting with all the CHWs. During the meeting the supervisor mentors the CHWs, reviews the CHWs performance and discusses with the CHWs the challenges they might be facing in their provision of community health services.
+
+{{< figure src="Monthly_meeting_workflows.png"  link="Monthly_meeting_workflows.png" alt="Monthly meeting workflows" title="" class="left col-10">}}
+
+### Quality monitoring workflow
+With the supervisor reference app, a supervisor can schedule a quality monitoring visit to assess the quality of services being provided by a CHW. During the quality monitoring visit a supervisor shadows CHWs as they provide services to household members. A supervisor can schedule additional quality monitoring follow up visits for CHWs who are identified to have some weak areas and may need further support and mentorship from the supervisor.
+
+{{< figure src="Qualtiy_monitoring_workflows.png"  link="Qualtiy_monitoring_workflows.png" alt="Qualtiy monitoring workflows" title="" class="left col-10">}}
+
+### Group session workflow
+
+{{< figure src="Group_session_workflows.png"  link="Group_session_workflows.png" alt="Group session workflows" title="" class="left col-10">}}
+
+### CHW Activity workflow
+A CHW activity task is generated for a supervisor if a CHW has been inactive for 3 months. This workflow enables a supervisor to follow up CHWs who have been inactive for three months and may need to be replaced.
+
+### Supervisor in-app targets
+A Supervisor has access to the analytics tab. The widgets on the in-app analytics tab enables a supervisor to view individual supervisor performance metrics and the supervisor progress towards the expected monthly goals. The supervisor also has access to CHW aggregate targets which will help the supervisor track metrics for individual CHWs, especially for supervisors overseeing a group of CHWs.This feature allows a supervisor to identify CHWs who may need to be supported actively.
+
+##### CHW aggregate targets
+{{< figure src="CHW_aggregate_targets.png"  link="CHW_aggregate_targets.png" alt="CHW aggregate targets" title="" class="left col-10">}}
+
+
+### Supervisor UHC mode
+A supervisor is able to use this feature to view the last time each CHW was visited so that a supervisor can prioritize visiting CHWs who have not been visited for a while. The visit detail information is updated every time a quality monitoring form is submitted.
+
+{{< figure src="Supervisor_UHC_mode.png"  link="Supervisor_UHC_mode.png" alt="Supervisor UHC mode" title="" class="left col-9">}}
```

Comment:
this image isn't using accurate in app screenshots.  Please redo in Figma or take them from a real app.

@n-orlowski can assist as needed